common/include/obc_assert.h

```diff
@@ -0,0 +1,9 @@
+#ifndef COMMON_INCLUDE_OBC_ASSERT_H_
+#define COMMON_INCLUDE_OBC_ASSERT_H_
+
+#define STATIC_ASSERT(COND, MSG) _Static_assert(COND, MSG)
+
+#define STATIC_ASSERT_EQ(A, B) STATIC_ASSERT((A) == (B), #A" does not equal "#B)
+#define STATIC_ASSERT_INT_EQ(A, B) STATIC_ASSERT((int)(A) == (int)(B), #A" does not equal "#B)
```

Comment:
For learning purposes, we could make one of the asserts part of the challenge to write? Understanding how to write it could help them appreciate how its used a little better

Reply:
Good idea, I'll add that